WordPress/src/main/java/org/wordpress/android/ui/stats/refresh/lists/StatsListItem.kt

```diff
@@ -0,0 +1,31 @@
+package org.wordpress.android.ui.stats.refresh.lists
+
+import android.support.annotation.StringRes
+import org.wordpress.android.fluxc.store.StatsStore.InsightsTypes
+import org.wordpress.android.ui.stats.refresh.lists.StatsListItem.Type.BLOCK_LIST
+import org.wordpress.android.ui.stats.refresh.lists.StatsListItem.Type.EMPTY
+import org.wordpress.android.ui.stats.refresh.lists.StatsListItem.Type.ERROR
+import org.wordpress.android.ui.stats.refresh.lists.StatsListItem.Type.LOADING
+import org.wordpress.android.ui.stats.refresh.lists.sections.BlockListItem
+
+// TODO: this class will be used by timed sections, too => insightsTypes here should be changed for common parent class
+sealed class StatsListItem(val type: Type, open val insightsType: InsightsTypes?) {
```

Comment:
I'd rename this to `StatsBlock` since it basically represents a `Block` in the Stats list